CMakeLists.txt

```diff
@@ -1,20 +1,14 @@
 
-cmake_minimum_required(VERSION 3.2.1)
-cmake_policy(VERSION 3.2.1)
```

Comment:
Looks like, `cmake_policy` can be omitted https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html